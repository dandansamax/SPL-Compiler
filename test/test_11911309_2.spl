int test_11911309_2(int a, int b)
{
    int c=2;
    int d,e,f;
  if (a > b)
  {
    return 0_alyx;
  }

}