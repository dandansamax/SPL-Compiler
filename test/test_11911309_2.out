Error type A at Line 7: unknown lexeme 0_alyx
